addons/mail/static/src/core/web/chat_window_service_patch.js

```diff
@@ -6,7 +6,11 @@ import { patch } from "@web/core/utils/patch";
 
 patch(ChatWindowService.prototype, {
     async _onClose(chatWindow, options) {
-        if (this.ui.isSmall && !this.store.discuss.isActive) {
+        if (
+            this.ui.isSmall &&
+            !this.store.discuss.isActive &&
+            (!chatWindow.thread || ["chat", "channel", "group"].includes(chatWindow.thread?.type))
```

Comment:
Condition on channel type are dangerous.
What should happen for `livechat` here? And `whatsapp`?

Reply:
This override of `_onClose()`is annoying because I don't want this click simulation to occur for livechats in mobile but I don't see how I could bypass this specifc override and still call `super`. My goal here was to exclude livechats using `type` but I unintentionally excluded whatsapp too... Maybe I could just rewrite it like: `chatWindow.thread?.type !== "livechat"` but it bothers me because "livechat" is not supposed to exist in the mail module.

Alternatively, I could add a layer to this override in the livechat module and trigger another click to close the messaging menu but that's a workaround on top of another workaround.